### ISSUE TITLE
Suggestion: get default value from initializer

### ISSUE THREAD

**Issue by buschtoens** [gossi/sparkles-decorators#1]
Just saw you addon in Ember Weekly. Nice!

Are you aware of [@ember-decorators/argument](https://github.com/ember-decorators/argument#argument)?

I like that the default value is provided as an initializer.

```ts
// src/ui/components/my-component/component.ts
import Component from 'sparkles-component';
import { arg } from 'sparkles-decorators';

export default class MyComponent extends Component {
  @arg foo = 'default value';
}
```

This has the added benefit of being able to drop the non-null assertion operator and letting TypeScript infer the type.

**Comment by gossi** [comment]
yup but couldn't use since doesn't work with TS yet.

What bogggles me about this `@arg foo = 'default value';` is, it doesn't look like it is bound to the `@arg`. It really just looks like a regular property with a value that doesn't receive updates form passed in arguments. The visually indistinctivity against other properties hindered me to use it so far, though can be easily implemented into.

At the end I see this merging into `@argument` when it will also support glimmer components. So the plan would be to bring both of them on track until then (that's why it is `@arg` and not `@argument`)

**Comment by gossi** [comment]
shipped in `0.0.2`
